cypress/support/a11y/checkA11y.js

```diff
@@ -0,0 +1,180 @@
+import { config as hasPageTitleCheck } from "./checks/has-page-title";
+import { config as pageTitleRule } from "./rules/page-title";
+
+/**
+ * add custom rules to our axe instance
+ *
+ * note: this is called automatically by `checkA11y`
+ *
+ * @param win window object of the current page
+ */
+const configureAxe = (win) => {
+  cy.configureAxe({
+    checks: [hasPageTitleCheck("MonComptePro", win)],
+    rules: [pageTitleRule("MonComptePro")],
```

Comment:
The `MonComptePro` string is just a copy/paste from the title set in the ejs layout, I guess we should put it somewhere else to prevent syncing issues?